src/styles/variables.less

```diff
@@ -58,42 +59,42 @@
 @shadow-inset: 0 10px 10px -10px @color-gray-30 inset;
 
 // default
-@color-default-borderColor: #c5c5c5;
-@color-default-backgroundColor: #ededed;
-@color-default-gradientStartColor: #f3f3f3;
-@color-default-gradientEndColor: #e2e2e2;
+@featured-color-default-borderColor: #c5c5c5;
+@featured-color-default-backgroundColor: #ededed;
+@featured-color-default-gradientStartColor: #f3f3f3;
+@featured-color-default-gradientEndColor: #e2e2e2;
 
 // primary
-@color-primary-borderColor: @color-primary;
-@color-primary-borderColorLite: fade(@color-primary, 40%);
-@color-primary-backgroundColor: fade(@color-primary, 30%);
-@color-primary-gradientStartColor: fade(@color-primary, 70%);
-@color-primary-gradientEndColor: @color-primary;
+@featured-color-primary-borderColor: @featured-color-primary;
+@featured-color-primary-borderColorLite: tint(@featured-color-primary, 60%);
```

Comment:
Nice use of tint! That should be our default unless we want the alpha that `fade` brings.